src/main/java/org/folio/circulation/domain/notice/schedule/ScheduledNoticesRepository.java

```diff
@@ -43,4 +52,42 @@ public ScheduledNoticesRepository(
       }
     });
   }
+
+  public CompletableFuture<Result<MultipleRecords<ScheduledNotice>>> findNoticesToSend(
```

Comment:
Please add sorting by nextRunTime to find notices created first

Reply:
Done